exdb/models.py

```diff
@@ -77,6 +76,15 @@ def needs_evaluation(self):
         return self.status == 'ad' and self.end_datetime <= now()
 
 
+class ExperienceApproval(models.Model):
+    experience = models.ForeignKey(Experience, related_name='approval_queue')
```

Comment:
I feel like `approval_set` would be a clearer name for this than `approval_queue`.